content/tools/eclipse/archive/EclipseTesting.md

```diff
@@ -0,0 +1,40 @@
+<!--
+title: "Testing on a Server Outside Eclipse"
+description: "Testing on a Server Outside Eclipse"
+tags: "tools eclipse testing"
+-->
+
+## Getting Started
+When the plugin first loads in Eclipse, it will deploy a Contrast Agent, a jar file beginning with the file name ***contrast-engine-***, to the user home directory under the hidden folder ***.contrast*** To monitor a server running outside of Eclipse, you will need this file. In the context below, ```${version}``` is the version number trailing this file name.  
```

Comment:
When the plugin is opened the first time, it will ask you to configure it with your TeamServer authentication details such as Username, API Key and Service Key. 
Vulnerability details are then retrieved from TeamServer and displayed in the Contrast Eclipse plugin.